.github/workflows/nix.yml

```diff
@@ -0,0 +1,93 @@
+name: Nix
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/nix.yml'
+      - 'nix/**'
+      - 'flake.*'
+  pull_request:
+    paths:
+      - '.github/workflows/nix.yml'
+      - 'nix/**'
+      - 'flake.*'
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-24.04
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v4
+
+      - name: Install Nix
+        id: install_nix
+        uses: nixbuild/nix-quick-install-action@v28
+
+      - name: Check Nix Flake
+        id: check
+        run: |
+          nix flake show
+          nix flake check
+
+      - name: Set matrix
+        id: set-matrix
+        run: |
+          # Create a "matrix" targeting the systems we want to check (and later deploy).
+          # It will be consumed by the next job(s) to fire one build per system in parallel.
+          # https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow
+          echo "matrix<<end_of_matrix" >> $GITHUB_OUTPUT
+          cat >> $GITHUB_OUTPUT <<-EOF
+          {"include":[
+          {"target":"webforge","hostname":"webforge","domain":"tahoe-lafs.org"}
+          ]}
+          EOF
+          echo "end_of_matrix" >> $GITHUB_OUTPUT
```

Comment:
What do the `matrix` and `end_of_matrix` markers do?  I'm not seeing them anywhere else in the code.

I searched the GHA docs another, not believing there's no better way, but came up black. :/

Reply:
`matrix` is the name of the output and `end_of_matrix` is a marker unique enough to tell GHA when the output value starts and stops.